CefSharp/Internals/JavascriptCallback.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright © 2010-2014 The CefSharp Project. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace CefSharp.Internals
+{
+    [DataContract]
+    internal sealed class JavascriptCallback : IJavascriptCallback
```

Comment:
To make things a little more `DRY`, should probably inherit `DisposableResource`.